src/environment-logs-provider.ts

```diff
@@ -28,13 +29,13 @@ export class EnvironmentLogsProvider {
     if (this.deploymentId) {
       this.logDeployment(this.deploymentId).catch((e) => {
         if (!axios.isCancel(e)) {
```

Comment:
why is the cancel check BTW? does it throw error? 🤔

Reply:
yes cancel throw an error, since we cancel on purpose we skip those errors.